.github/ISSUE_TEMPLATE/submission.yml

```
@@ -45,8 +45,7 @@ body:
       description: >
         1-2 sentences to appear under your project on
         [qiskit.github.io/ecosystem](https://qiskit.github.io/ecosystem).
-        Keep it under 30 words; readers will click through to your repo if
-        they want to learn more.
+        Keep it under 135 characters; anything longer will be trimmed.
```

Trimmed text looks bad. I think we should reject the submission until it's fixed:

```suggestion
        Keep it under 135 characters; anything longer will be rejected.
```

Could you please add a test that all entries have descriptions <135?

---

Good idea!